File: respostas/C/Beginner/1009/main.c
#include <stdio.h>
#include <stdlib.h>
 
int main() {
 
    char *name = (char*) malloc(sizeof(char) * 15);
    scanf(" %[^\n]", name);
    
    double fixedSalary, sales, salary;
    scanf("%lf%lf", &fixedSalary, &sales);
    
    salary = fixedSalary + sales*0.15;
    printf("TOTAL = R$ %.2f\n", salary);
 
    return 0;
}

File: respostas/Java/Beginner/1009/Main.java
import java.util.Scanner;

class Main {

    public static void main(String args[]) {
        Scanner scan = new Scanner(System.in);

            String name = scan.nextLine();
            double fixedSalary = scan.nextDouble(), total = scan.nextDouble(), finalSalary;

            finalSalary = fixedSalary + (total * 0.15);

            System.out.printf("TOTAL = R$%.2f\n", finalSalary); 

        scan.close();
    }

}